src/patternfly/components/Table/table.scss

```
@@ -1064,6 +1077,11 @@
     --#{$table}--cell--PaddingBlockEnd: var(--#{$table}--m-compact__action--PaddingBlockEnd);
   }
 
+  .pf-m-action {
```

Can you tie this modifier class to the table element it applies to?

```suggestion
  .#{$table}__td.pf-m-action {
```